src/NumericInput.js

```diff
@@ -49,6 +49,12 @@ export default class NumericInput extends PureComponent {
         this.props.onChange(e);
     }
 
+    componentWillUpdate(nProps, nState){
+        if (this.state.value.length - nState.value.length === 2) {
```

Comment:
Why 2?
What if `0.` vs `0.1` ?

and why append 0 to nState.value works?
write some comments as it is non-obvious?